starcheck/calc_ccd_temps.py

```diff
@@ -152,8 +152,19 @@ def get_ccd_temps(oflsdir, outdir='out',
 
     # Get tstart, tstop, commands from backstop file in opt.oflsdir
     bs_cmds = get_bs_cmds(oflsdir)
-    tstart = DateTime(bs_cmds[0]['date']).secs
-    tstop = DateTime(bs_cmds[-1]['date']).secs
+
+    # Use RLTT and SCHEDULED_STOP_TIME if available
+    rltt = bs_cmds['event_type'] == 'RUNNING_LOAD_TERMINATION_TIME'
```

Comment:
Always use `ok` per Ska idioms in this situation.  This tells the reader right away that it is a temporary selection mask.